2020-01-08 13:41:15,492 - D:\python_project\zidonghua\TestCase\test01case.py[line:53) - DEBUG:登录成功]
